Packages/Sites/Sfi.Kateheo/Resources/Private/TypoScript/NodeTypes/TagsPage.ts2

```diff
@@ -0,0 +1,26 @@
+prototype(Sfi.Kateheo:TagsPage) {
```

Comment:
What are you trying to show on this page? How is it different from `Sfi.Kateheo:Tags`?